entrypoint.sh

```diff
@@ -43,6 +43,16 @@ elif [ -n "$NGROK_HOSTNAME" ] || [ -n "$NGROK_SUBDOMAIN" ]; then
   fi
 fi
 
+# Set the remote-addr if specified
+if [ -n "$NGROK_REMOTE_ADDR" ] && [ -n "$NGROK_AUTH" ]; then
+  ARGS="$ARGS -remote-addr=$NGROK_REMOTE_ADDR "
+elif [ -n "$NGROK_REMOTE_ADDR" ]; then
+  if [ -z "$NGROK_AUTH" ]; then
+    echo "You must specify an authentication token after registering at https://ngrok.com to use reserved ip addresses."
+    exit 1
+  fi
```

Comment:
I'd rewrite it so:

if [ -n "$NGROK_REMOTE_ADDR" ]; then
  if [ -z "$NGROK_AUTH" ]; then
    echo "You must specify an authentication token after registering at https://ngrok.com to use reserved ip addresses."
    exit 1
  fi
  ARGS="$ARGS -remote-addr=$NGROK_REMOTE_ADDR "
fi